Dockerfile

```diff
@@ -0,0 +1,33 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM centos:centos7
+LABEL maintainer="Presto Software Foundation <prestosql.io>"
```

Comment:
In docker-images we use
```
MAINTAINER Presto community <https://prestosql.io/community.html>
```
`MAINTAINER` directive is obsolete, so I'd use
```
LABEL maintainer="Presto community <https://prestosql.io/community.html>"
```
here

Reply:
Done.